src/sentry/event_manager.py

```diff
@@ -691,8 +691,9 @@ def save(self, project, raw=False):
             ), timestamp=event.datetime)
 
         if is_new and release:
-            buffer.incr(Release, {'new_groups': 1}, {
-                'id': release.id,
+            buffer.incr(ReleaseProject, {'new_groups': 1}, {
```

Comment:
We need to figure out how to handle this for rows that are NULL. Since incrementing a NULL value results in NULL.

I'm not sure the best answer for this besides changing `buffer.process` to run some raw SQL. to properly do it.